### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by DronRathore**
@debloper : Do not merge for time being, I will let you know when all of them which are on list gets done.

### CODE REVIEW

**Comment by DronRathore** [comment]
@debloper : Merge.

**Comment by DronRathore** [comment]
Will have to copy all the assets for future run of 3rd parties(i.e. logo), also @vineelreddy Do we have any updates for sponsors Or are they all the same?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 mozHive
 =======
+This code is distributed under MPL(Mozilla Public License).
```
Comment (debloper) [inline README.md]: MPL version?

**Comment by debloper** [comment]
Small fix required to update the MPL version & the code indentation & spacing in `front-end/squash.js`

**Comment by DronRathore** [comment]
@debloper Done :smile:

**Comment by debloper** [comment]
Are `@-o-keyframes, @-ms-keyframes` really required?
Anyway, didn't verify locally, going ahead and merging, as it works for you. :+1:

**Comment by DronRathore** [comment]
Just added for backward support, have you deployed it yet?

**Comment by debloper** [comment]
I don't think `@-ms-keyframes` was actually ever implemented (MS went with non-prefixed version, even while nto feature complete) or, `@-o-keyframes` was actually required (cause it falls back on standard declaration in *really* old Opera, and it works; for newer version, it expects `-webkit-` prefix).

The changes are live, you may have to hard refresh, just in case of caching.

**Pull request merged**